pkg/services/zulip/zulip_config.go

```diff
@@ -0,0 +1,68 @@
+package zulip
+
+import (
+	"errors"
+	"net/url"
+)
+
+// Config for the zulip service
+type Config struct {
+	BotMail string
+	BotKey  string
+	Host    string
+	Path    string
+	Stream  string
+	Topic   string
+}
+
+// GetURL returns a URL representation of it's current field values
+func (config *Config) GetURL() *url.URL {
+	return &url.URL{
+		User:   url.UserPassword(config.BotMail, config.BotKey),
+		Host:   config.Host,
+		Path:   config.Path,
+		Scheme: Scheme,
```

Comment:
The Query parameters are never set (`Stream` and `Topic`)

Reply:
You are right. Fixed. Thank you.